.circleci/config.yml

```diff
@@ -641,26 +658,26 @@ jobs:
           command: make V=1 J=8 -j8 jtest
       - post-steps
 
-  build-linux-java-pmd:
-    machine:
-      image: ubuntu-2004:202111-02
-    resource_class: large
-    environment:
-      JAVA_HOME: /usr/lib/jvm/java-8-openjdk-amd64
-    steps:
-      - install-maven
-      - pre-steps
-      - run:
-          name: "Set Java Environment"
-          command: |
-            echo "JAVA_HOME=${JAVA_HOME}"
-            echo 'export PATH=$JAVA_HOME/bin:$PATH' >> $BASH_ENV
-            which java && java -version
-            which javac && javac -version
-      - run:
-          name: "PMD RocksDBJava"
-          command: make V=1 J=8 -j8 jpmd
-      - post-pmd-steps
+#  build-linux-java-pmd:
+#    machine:
+#      image: ubuntu-2004:202111-02
+#    resource_class: large
+#    environment:
+#      JAVA_HOME: /usr/lib/jvm/java-8-openjdk-amd64
+#    steps:
+#      - install-maven
+#      - pre-steps
+#      - run:
+#          name: "Set Java Environment"
+#          command: |
+#            echo "JAVA_HOME=${JAVA_HOME}"
+#            echo 'export PATH=$JAVA_HOME/bin:$PATH' >> $BASH_ENV
+#            which java && java -version
+#            which javac && javac -version
+#      - run:
+#          name: "PMD RocksDBJava"
+#          command: make V=1 J=8 -j8 jpmd
+#      - post-pmd-steps
```

Comment:
So we don't need this stage any more, right?

Reply:
Yes, unless we add the maven-pmd-plugin related parts in `pom.xml.template`.
we can add it in the following commit if necessary. 
Right now , i think it's no needed.